src/utils/available-rpc.ts

```diff
@@ -68,13 +68,13 @@ const getBlockNumber = async (
 
   // Conditionally handle what type of provider is being passed
   let rpcProvider: Provider;
-
-  // If URL starts with wss, use WebSocketProvider
   if(provider.startsWith('wss')) {
-    rpcProvider = new WebSocketProvider(provider, network);
+    rpcProvider = new WebSocketProvider(provider, network, {
+      staticNetwork: network, // Network is dictated in the RPC URL, will not change
```

Comment:
silly q but do we need this comment at all ? since its documented inside ethers 

```
 *  **``staticNetwork``** - do not request chain ID on requests to
 *  validate the underlying chain has not changed (default: ``null``)
```

Reply:
I personally prefer more comments always to save time for anyone dealing with it in the future